python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(cls.gangadir, ignore_errors=True)
```

Comment:
Frankly I'd rather keep it as it should work bar a file remove command not working.
This isn't failing to get the correct arguments to shutil.rmtree so I would like to keep this for the time being